app/scripts/lib/backend-metametrics.js

```diff
@@ -1,9 +1,7 @@
 import { getBackgroundMetaMetricState } from '../../../ui/app/selectors'
 import { sendMetaMetricsEvent } from '../../../ui/app/helpers/utils/metametrics.util'
 
-const inDevelopment = process.env.NODE_ENV === 'development'
```

Comment:
Oops - this was missed in #9093. I hadn't realized this was being set in two places. So we're still using the production tracking URL for metrics sent in tests.

Thankfully we're sending to the correct project still, which is the most important thing. Honestly I don't know what this tracking URL is for, and I suspect it might be completely unnecessary. But regardless I'll fix this in a separate PR.